src/constants/transactionTypes.js

```diff
@@ -5,6 +5,13 @@ export default {
   vote: 3,
 };
 
+export const transactionNames = {
+  0: 'Send',
+  1: 'Second passphrase registration',
+  2: 'Delegate registration',
+  3: 'Delegate vote',
```

Comment:
This is duplicating transaction names in https://github.com/LiskHQ/lisk-hub/blob/baae7626cd59f4105c739ebf19fd357fabe8f9cb/src/components/screens/wallet/transactions/address/TransactionAddress.js#L16-L33

It would be ideal to move that component to `/shared` and reuse it.

Reply:
I think it definitely makes sense to move `TransactionAddress` to `shared`, since it is being used both in `dashboard` and `wallet`. However those styles are a bit different than the ones being used in the `TransactionsTable`. On the `TransactionAddress` component the LSK address is not truncated and the bookmarked accounts are displayed with the address as a subtitle.
What do you think if I move the `TransactionAddress` component to `/shared` and remove the switch statement there so that it simply takes the string from the `transactionTypes` constants file?

Reply:
Sounds good. 

There's just one problem you'll face, when you'll move all transaction type labels to the const file - translation `t(...)` function

- If you don't wrap all transaction type labels with `t` in the const file, the scanner will no longer find them, and will remove them from `common.json`
- If you do wrap all transaction type labels with `t` in the const file, it will evaluate only once and won't update on language change.

There are some hacks around it to choose from. In Hub codebase, we have:
https://github.com/LiskHQ/lisk-hub/blob/development/src/constants/networks.js#L24-L25
or 
https://github.com/LiskHQ/lisk-hub/blob/55e5b7bfbf3905bba0fa988572a4460cba8f9c96/src/constants/quickTips.js#L7

Reply:
Thank you Vit, changes implemented.